examples/simple/README.md

```diff
@@ -0,0 +1,33 @@
+
+# Simple Trace Example
+
+This example sets up a tracer in the main application then makes several calls to a custom library instrumented with the <a href="https://github.com/open-telemetry/opentelemetry-cpp"> Open Telemetry SDK </a>. Specifically, it demonstrates the setup of fundamental Open Telemetry componenets such as Tracers, Tracer Providers and Spans.  The example also illustrates context propogation from parent spans to children spans and events.  All telemetry output is directed to stdout.
```

Comment:
I love the idea of having a README for the example. I'd prefer being more concise here. The most important things to point out:
* There's a `foo_library` that is instrumented with the OpenTelemetry API.
* There's an application in `main.cc` that initializes and registers a tracer provider from the OpenTelemetry SDK. Telemetry data is directed to stdout by using a custom exporter.